graph/pull_v2.go

```diff
@@ -223,6 +223,9 @@ func (p *v2Puller) pullV2Tag(tag, taggedName string) (verified bool, err error)
 	go func() {
 		if _, err := io.Copy(out, pipeReader); err != nil {
 			logrus.Errorf("error copying from layer download progress reader: %s", err)
+			if _, err := io.Copy(ioutil.Discard, pipeReader); err != nil {
```

Comment:
Can you explain a bit more. Afaik this `pipeWriter` is directly thrown to the `download` and from there to `progressreader`. I guess we could also just close `pipeReader` here(cleaner but errors for writers) but not not sure how we do it from `download`. Just so that everything is clear, we are not actually canceling the pull, we just want the progress writers not to block.